### PR TITLE
Prefix the build with appropriate folder

### PR DESCRIPTION
**Pull request by Ryman**
This ensures that js and links for the PR builds are working and are not reliant on whatever exists on master.

Known issue: favicons are broken on PR builds, but seems like an easy followup

### CODE REVIEW

**Comment by moekify** [comment]
Amen

**Comment by Ryman** [comment]
QA: http://staging.honeypot.co/pr-111/ (pending build)

File: gatsby-config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ require('dotenv').config({
 })
 
 module.exports = {
-  pathPrefix: process.env.HOST,
+  pathPrefix: process.env.PREFIX_FOLDER,
```
Comment (lean8086) [inline gatsby-config.js:9]: @Ryman should we use `URL_BASE` instead?

Comment (Ryman) [reply gatsby-config.js:9]: Oh I missed that var, I guess we could but I'm unsure if the `/` will mess things up. I'll merge this for now but sounds worth trying in a followup to simplify? Or should we be removing the URL_BASE? do you know what it is used for, a search shows nothing?

**Pull request merged**